src/data/blog/Hiring-web-dev-092019.md

```diff
@@ -26,44 +26,59 @@ author_facebook: null
+Alongside your core technical skills, you’ll also need to be:
 
-- Be comfortable communicating in a remote team;
-- Enjoy taking part in conversations in technical subjects;
-- Be able to demonstrate that you enjoy expanding your knowledge and thinking critically (this can be via a degree, but 1/3rd of our team members don’t have one, so doesn’t have to be);
-- Value agility in software development (experience of working in an agile environment is an advantage but not essential)
-- Be passionate about accessibility
-- Be keen to work in an organisation that delivers both commercial and social good projects
-- Be inspired by user-centred development (even if you don’t have practical experience of this approach)
-- Be committed to taking part in team activities and individual learning
+- comfortable communicating in a remote team
+- happy taking part in conversations about technical subjects
+- able to demonstrate that you enjoy expanding your knowledge and thinking critically (this can be via a degree, but 1/3rd of our team members don’t have one, so doesn’t have to be)
+- value agility in software development (experience of working in an agile environment is an advantage but not essential)
+- passionate about accessibility
+- keen to work in an organisation that delivers social good projects
+- inspired by user-centred development (even if you don’t have practical experience of this approach)
 
 ##What this role is not:  
   
-This is not a beginner role. You’ll need to have worked in coding teams on established projects and have well-developed skills in the appropriate languages. You’ll also need to be able to show how your work experience to date equips you both for communicating with a remote team and getting your head down to get on with the task.
+This is not a beginner role. You’ll need to have worked in coding teams on established projects and have some experience in the appropriate languages. You’ll also need to be able to show how your work to date equips you both for communicating with a remote team and getting on with the task at hand.
 
 ##What this role is:  
   
-A great opportunity for growth and enjoyment. Neontribe prides itself on skills development, great training budgets, flexible working, part-time hours and supporting return to work and other initiatives to help you enjoy your work. You’ll be stepping into the shoes of a developer who taught herself to code while she was working in retail and banking and then spent 3 years at Neontribe before heading to London as a front end developer for a major fashion brand.
+A great opportunity for growth and enjoyment. Neontribe prides itself on skills development, great training budgets, flexible working, part-time hours and supporting return to work and other initiatives to help you enjoy your work. 
 
 ##Salary  
 
-We’d expect to pay in the region of £24,786.00 - £29,532.00 for this role.
+We’d expect to pay in the region of £35,000.00 - £42,000.00 for this role.
 
 To Apply  
```

Comment:
Should this be a heading?

Reply:
It Is A hEaDiNg. It HaS aLwAyS bEeN a HeAdInG...

Reply:
It's totally a Heading. It was always a headling. I don't know what you mean....

(Fixed.)